Slim/Plugin/Deezer/strings.txt

```diff
@@ -106,32 +106,39 @@ PLUGIN_DEEZER_SKIPS_EXCEEDED
 
 PLUGIN_DEEZER_SMART_RADIO
 	EN	Deezer Smart Radio
+	FR	Deezer Smart Radio
 
 PLUGIN_DEEZER_FLOW
 	DE	Deezer Flow (nur Premium+ Nutzer)
 	EN	Deezer Flow (Premium+ users only)
-	FR	Deezer Flow (Premium+ uniquement)
+	FR	Deezer Flow
```

Comment:
Please add the note about the availability, unless it's not restricted any more. In which case I could remove it from the other languages.

Reply:
Personally, I don't have a Deezer account to verify.
This page made me doubt: https://support.deezer.com/hc/fr/articles/201331091-Deezer-et-Musique-
I am going back to the initial text.